tests/misc/CachingTest.py

```diff
@@ -74,6 +74,22 @@ def test_get_uncached_files(self, mock_os):
         cache.current_time = 2
         self.assertEqual(cache.get_uncached_files({file_path}), set())
 
+    def test_reset_cache_on_bear_error(self):
+        all_files = {'file.c'}
+        cache = FileCache(self.log_printer, 'coala_test2', flush_cache=True)
+        cache.track_files(all_files)
+
+        loaded_local_bears_count = 3
+        loaded_valid_local_bears_count = 2
+
+        if(loaded_local_bears_count != loaded_valid_local_bears_count):
+            if cache:
+                cache.untrack_files(all_files)
+
+        cache.write()
+        cache = FileCache(self.log_printer, 'coala_test2', flush_cache=False)
+        self.assertFalse('file.c' in cache.data)
```

Comment:
don't understand the test. You've copied the code which is actually not related to the `Cache` itself^^ you're not testing the implementation inside Processing.py